telemetry/definitions/commonDefinitions.json

```diff
@@ -2601,15 +2601,41 @@
             "unit": "Count"
         },
         {
-            "name": "awsq_thumbsUp",
+            "name": "awsq_codeGenerationThumbsUp",
             "description": "User clicked on the thumbs up button, to mention that they are satisfied",
             "unit": "Count"
```

Comment:
Do we increment this count corresponding clientId/userId each time user click thumps up? or does this count correspond to some sessions/duration? 

e.g. user clicked thumps up 3 times in 5 mins so `count:3 ` suggests per client/ 5 mins?